run.sh

```diff
@@ -99,7 +99,7 @@ function tear_down() {
 }
 
 function run_tests() {
-   eval "./mvnw -Dtest=$TESTS  test"
+   eval "./mvnw -Dtest=$TESTS -DPLATFORM_TYPE=$PLATFORM test"
```

Comment:
I added it as a -D parameter since I want the value to be passed in even when not running the setup scripts, there is no init script run all the time